### ISSUE TITLE
.md file extension support

### ISSUE THREAD

**Issue by Locutus73** [MiSTer-devel/FpgaGen_MiSTer#1]
Can you please add support for .md (not .smd) extension to FpgaGen? It’s the same rom binary dump format than .gen with a different extension in order to specify a Mega Drive dump (i.e. you can rename .md files to .gen and load them successfully in FpgaGen).

**Comment by sorgelig** [comment]
If it's just different extension with the same content, then i will add it.

**Comment by Locutus73** [comment]
Yes, I confirm that: same exact content, just different extension.

Thank you
